prime-dotnet-webapi/Startup.cs

```diff
@@ -75,8 +75,9 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<IHealthAuthoritySiteService, HealthAuthoritySiteService>();
             services.AddScoped<ILookupService, LookupService>();
             services.AddScoped<IMetabaseService, MetabaseService>();
-            services.AddScoped<IOrganizationService, OrganizationService>();
             services.AddScoped<IOrganizationAgreementService, OrganizationAgreementService>();
+            services.AddScoped<IOrganizationClaimService, OrganizationClaimService>();
+            services.AddScoped<IOrganizationService, OrganizationService>();

```

Comment:
I decided to order the services alphabetically assuming the word "Service" is not included; the suffix "service" is a convention and I personally think about it more like "Organization Service" than an "OrganizationService" if that makes any sense.  Sounds a bit crazy when I write it out that way 😛.

This also has the side effect of "bubbling up" the "base" service to the top of the group:

DocumentService
DocumentAccessTokenService

EmailService
EmailDocumentsService
EmailRenderingService
EmailTemplateService

EnrolleeService
EnrolleeAgreementService
EnrolleePaperSubmissionService
EnrolleeSubmissionService

HealthAuthorityService
HealthAuthoritySiteService

OrganizationService
OrganizationAgreementService
OrganizationClaimService

SubmissionService
SubmissionRulesService

But if it makes more sense to you to do a strict alphabetical ordering I can see the merit.

Reply:
Just thinking it's easier for someone to keep them ordered by e.g. copying and pasting into NotePad++ and doing a Edit > Line Operations > Sort ...

Reply:
Added comment re: ordering convention